easy_my_coop_payment_term/models/company.py

```diff
@@ -0,0 +1,12 @@
+from odoo import api, fields, models
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    default_payment_term = fields.Many2one(
+            "account.payment.term",
+            company_dependent=True,
+            string="Default Payment Term",
+            help="Default payment term to use when" 
+            "creating capital release request invoices",
+        )
```

Comment:
I think you forgot to run pre-commit. The indentation is off.

Reply:
Nice catch, thank you :)